Gemfile

```diff
@@ -26,6 +26,8 @@ gem "turbolinks", "~> 5"
 gem 'devise'
 gem 'carrierwave', '~> 1.0'
 gem 'foundation-rails'
+gem 'rails-controller-testing'
+
```

Comment:
Excess line?

Reply:
Yung luma na to sir natanggal ko na yan Hahaha